### PR TITLE
Initial implementation of edit policy page

### PR DESCRIPTION
**Pull request by ycao56**
Initial implementation of edit policy page https://github.com/open-cluster-management/backlog/issues/11168
unit test and code refactoring to do
image
image
image

### CODE REVIEW

File: scss/common.scss [hunk 1]
```diff
@@ -84,30 +83,8 @@ body {
...
     left: 50%;
   }
-  // .bx--select--inline{
-  //   svg.bx--select__arrow{
-  //     position: absolute
-  //   }
-  // }
-}
-
-#toggle-perspective {
```
Comment (willkutler) [inline scss/common.scss:94]: does removing this not mess up the style for the perspective switcher?

File: src-web/components/common/TemplateEditor/scss/template-editor.scss [hunk 2]
```diff
@@ -25,29 +25,16 @@
...
-    }
-
-    .creation-view-controls-container {
-      margin-top: 30px;
-    }
-
     &.showEditor {
-      height: calc(100vh - 190px);
-
+      // height: calc(100vh - 190px);
```
Comment (willkutler) [inline src-web/components/common/TemplateEditor/scss/template-editor.scss:29]: can these comments be removed?

**Review by willkutler — CHANGES_REQUESTED** [review]

File: scss/common.scss (hunk 1 above)
Comment (ycao56) [reply scss/common.scss:94]: will remove in future clean up pr

File: src-web/components/common/TemplateEditor/scss/template-editor.scss (hunk 2 above)
Comment (ycao56) [reply src-web/components/common/TemplateEditor/scss/template-editor.scss:29]: will remove in future clean up pr

File: scss/common.scss (hunk 1 above)
Comment (ycao56) [reply scss/common.scss:94]: I didn't know this is for toggle-perspective switcher. Shouldn't it be in header instead?

Comment (willkutler) [reply scss/common.scss:94]: this is in header, it's just that something that I couldn't find in the grc style was overriding it so I added that to fix it. If the perspective switcher looks ok after removing it maybe it's not necessary though

**Review by dhaiducek — APPROVED** [review]
This is great. I assume future enhancements will be to not prompt the user to update the policy? Or if they do something like change the name, ask if they want to create a new policy (or make immutable fields un-editable)?

File: src-web/components/common/TemplateEditor/scss/template-editor.scss (hunk 2 above)
Comment (ycao56) [reply src-web/components/common/TemplateEditor/scss/template-editor.scss]: I updated the patternfly css to latest so in theory we should no longer need to override css. Can you test it again after the pr is merged? It it doesn’t work, we can add css back

File: scss/common.scss (hunk 1 above)
Comment (ycao56) [reply scss/common.scss:94]: I updated the patternfly css to latest so in theory we should no longer need to override css. Can you test it again after the pr is merged? It it doesn’t work, we can add css back

Comment (ycao56) [reply scss/common.scss:94]: It could be also carbon components overriding the css. How can I test it?

**Pull request merged**